examples/template.html

```diff
@@ -19,117 +19,29 @@
         width: 100vw;
         margin: 0;
       }
+      #{{ plot_id }} {
```

Comment:
I meant the class defined above this class, is that really needed? 

Yoshiki Vázquez-Baeza

> On Apr 21, 2016, at 3:38 PM, Joshua Shorenstein <notifications@github.com> wrote:
> 
> In examples/template.html:
> 
> > @@ -19,117 +19,29 @@
> >          width: 100vw;
> >          margin: 0;
> >        }
> > +      #{{ plot_id }} {
> This is needed to get the resizing to work properly. Without it, the height stays at 500px regardless of what we do.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
>